packages/core/src/internal-components/popup/popup.element.ts

```diff
@@ -52,7 +52,7 @@ import { CdsInternalPointer } from './pointer.element.js';
  * </cds-internal-popup>
  * ```
  *
- * @beta
+ * @internal
```

Comment:
Popup is part of dropdowns. It should stay beta in 6.x.

Reply:
reverted